changelog/21.0/21.0.0/summary.md

```diff
@@ -33,4 +34,36 @@ The following metrics that were deprecated in the previous release, have now bee
 |      `reparent_shard_operation_timings`      |  
 		
 
+### <a id="traffic-mirroring"/>Traffic Mirroring
 
+Traffic mirroring is intended to help reduce some of the uncertainty inherent to `MoveTables SwitchTraffic`. When traffic mirroring is enabled, VTGate will mirror a percentage of traffic from one keyspace to another.
+
+Mirror rules may be enabled through `vtctldclient` in two ways:
+
+  * With `ApplyMirrorRules`.
+  * With `MoveTables MirrorTraffic`, which uses `ApplyMirrorRules` under the hood.
```

Comment:
Why do we need two ways of doing this?

Reply:
hm..technically no, but i would say it's similar to the relationship between `SwitchTraffic --tablet_types replica` and `ApplyRoutingRules`. The former is mainly an easier way of doing that latter.

in the case of `MoveTables MirrorTraffic` and `ApplyMirrorRules`, the former is the easiest way to use the latter, but the latter can also be used for advanced use cases.

Reply:
The difference is that with SwitchTraffic - creating routing rules is a part of the flow, but not all of it (not even the most important), whereas the two ways of starting mirroring seem to be identical. I'm thinking of future maintenance, so maybe @rohit-nayak-ps should chime in here.

Reply:
yeah that makes sense to me. i'm happy to remove it the API while still leaving the plumbing in place that `MirrorTraffic` is relying on. wouldn't be hard to add back in the future if demand for it arises.

Reply:
@deepthi i went a head and removed it